www/src/components/profile/Icon.tsx

```diff
@@ -0,0 +1,57 @@
+import { TrashCanIcon } from 'pluralsh-design-system'
+import { ReactNode, forwardRef } from 'react'
+import { DivProps, Flex } from 'honorable'
+import { useTheme } from 'styled-components'
+
+type Hue = 'none' | 'default' | 'lighter' | 'lightest'
+
+const hueToHoverBG: Record<Hue, string> = {
+  none: 'fill-zero-hover',
+  default: 'fill-one-hover',
+  lighter: 'fill-two-hover',
+  lightest: 'fill-three-hover',
+}
+
+type IconProps = DivProps & {
+  icon: ReactNode
+  hue?: Hue
+  hover?: string
+}
+
+export const Icon = forwardRef<HTMLDivElement, IconProps>(({
```

Comment:
This looks like a DS component candidate.

Reply:
For sure I can pull that out into the design system once this is all done.